### DISCUSSION TITLE
I have done my 13  E-commerce scans.but I don't have any port token in my zilpay wallet ..why...it's scam..

### DISCUSSION THREAD

**Discussion by harishchauhan911** [bearbywallet/Bearby#22] [Q&A]
I have done my 13  E-commerce scans.but I don't have any port token in my zilpay wallet ..why...it's scam..

**Comment by Debojyati-ux** [comment]
Yes I also scanned barcodes 5-6 times but i didn't get any port till now

**Reply by ANKIT4005** [reply]
I am also scan 7 time but I did not receive any port now?
